src/components/atoms/Checkbox/Checkbox.scss

```diff
@@ -0,0 +1,108 @@
+@import "scss/constants";
+
+$checkbox-diameter: 32px;
+
+$checkbox-transition: all 200ms cubic-bezier(0.23, 1, 0.32, 1);
+$checkbox-shadow-preset: inset 0 0 0 1px;
+
+$toggler-width: 56px;
+$toggler-height: 30px;
+
+$toggler-circle-diameter: 26px;
+$toggler-circle-spacing: 2px;
+
+$toggler-circle-moving-delta: $toggler-width - $toggler-circle-diameter -
+  $toggler-circle-spacing;
+
+.Checkbox {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  position: relative;
+
+  cursor: pointer;
+
+  // unsetting global label styles
+  margin: 0;
+
+  &__native-input {
+    &:checked + .Checkbox__custom-input {
+      background-color: $primary;
+
+      .Checkbox__tick-icon {
+        opacity: 1;
+        transform: scale(1);
+      }
+
+      &--toggler {
+        background-color: $dimmer-primary;
+
+        &::after {
+          left: $toggler-circle-moving-delta;
+
+          background-color: $primary;
+        }
+      }
+    }
+  }
+
+  &__custom-input {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: $checkbox-diameter;
+    height: $checkbox-diameter;
+
+    box-shadow: $checkbox-shadow-preset grey-color(0.1);
+    background-color: grey-color(0.12);
+
+    border-radius: $border-radius--max;
+
+    &:hover {
+      background-color: grey-color(0.16);
+      box-shadow: $checkbox-shadow-preset grey-color(0.4);
+    }
+
+    .Checkbox__tick-icon {
+      opacity: 0;
+      transform: scale(0.2);
+      transition: $checkbox-transition;
+    }
```

Comment:
The styles you highlighted are the basic/default ones

Reply:
Why are they defined within a different BEM `__element` as overrides then?

Reply:
I'd say it can be more explicit to have it there given the `overrides`

![image](https://user-images.githubusercontent.com/63194656/116928513-daf38980-ac65-11eb-91e1-3b5a9c7ddc3e.png)

It's neither possible to move Checkbox from `--toggler`  nor from ::checked state

![image](https://user-images.githubusercontent.com/63194656/116928664-070f0a80-ac66-11eb-9113-c10f784ef3ab.png)

Reply:
> It's neither possible to move Checkbox from --toggler nor from ::checked state

That seems fine though? As that would be our standard pattern of overrides yeah?

> I'd say it can be more explicit to have it there given the overrides

Whereas this part confuses me, since you're saying it is the default styles.. so I don't understand why we would change from our standard pattern for how we represent default styles for a BEM `__element` (eg. defined at the 'root' of the BEM class as `&__element`)

Am I missing something?

Reply:
done in https://github.com/sparkletown/sparkle/pull/1299/commits/fa746a0d247b163eae4a4127a1228bd96131c4a2